### ISSUE TITLE
E3001 Invalid or unsupported Type AWS::Glue::DataCatalogEncryptionSettings for resource GlueCatalogEncryptionSettings in us-gov-west-1

### ISSUE THREAD

**Issue by seanlogan-wh** [aws-cloudformation/cfn-python-lint#1855]
*cfn-lint version: cfn-lint 0.44.2*

*Description of issue.*
`E3001 Invalid or unsupported Type AWS::Glue::DataCatalogEncryptionSettings for resource GlueCatalogEncryptionSettings in us-gov-west-1.`  Confirmed it is in the spec for us-gov-east-1.  Just missing in corresponding west region.

Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.

--- 
```yaml
  GlueCatalogEncryptionSettings:
    Type: AWS::Glue::DataCatalogEncryptionSettings
    Properties: 
      CatalogId: !Ref AWS::AccountId
      DataCatalogEncryptionSettings:
        EncryptionAtRest: 
          CatalogEncryptionMode: SSE-KMS
          SseAwsKmsKeyId: !Ref KMS
```

*If present, please add links to the (official) documentation for clarification.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-datacatalogencryptionsettings.html
*Validate if the issue still exists with the latest version of cfn-lint and/or the latest Spec files
Confirmed that issue persists after updating cfn-lint to 0.44.2 and running cfn-lint --update-specs.

**Comment by PatMyron** [comment]
[`AWS::Glue::DataCatalogEncryptionSettings`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-datacatalogencryptionsettings.html) now in the [`us-gov-west-1` Resource Specification](https://s3.us-gov-west-1.amazonaws.com/cfn-resource-specifications-us-gov-west-1-prod/latest/CloudFormationResourceSpecification.json). Should be in the next release or through running `cfn-lint --update-specs` already. Sorry about the delay
